x-pack/plugins/security_solution/public/common/utils/route/index.test.tsx

```diff
@@ -9,64 +9,36 @@ import { mount } from 'enzyme';
 import React from 'react';
 
 import { HostsTableType } from '../../../hosts/store/model';
-import { RouteSpyState } from './types';
 import { ManageRoutesSpy } from './manage_spy_routes';
 import { SpyRouteComponent } from './spy_routes';
 import { useRouteSpy } from './use_route_spy';
-
-type Action = 'PUSH' | 'POP' | 'REPLACE';
-const pop: Action = 'POP';
-
-const defaultLocation = {
-  hash: '',
-  pathname: '/hosts',
-  search: '',
-  state: '',
-};
-
-export const mockHistory = {
-  action: pop,
-  block: jest.fn(),
-  createHref: jest.fn(),
-  go: jest.fn(),
-  goBack: jest.fn(),
-  goForward: jest.fn(),
-  length: 2,
-  listen: jest.fn(),
-  location: defaultLocation,
-  push: jest.fn(),
-  replace: jest.fn(),
-};
-
-const dispatchMock = jest.fn();
-const mockRoutes: RouteSpyState = {
-  pageName: '',
-  detailName: undefined,
-  tabName: undefined,
-  search: '',
-  pathName: '/',
-  history: mockHistory,
-};
+import { generateHistoryMock, generateRoutesMock } from './mocks';
```

Comment:
This test file was exporting a mock, which was being imported by other tests in a different module. The import itself would cause these test here to also run and i was getting weird errors. So I moved the mocks to a separate module to avoid importing it from a test file